great_expectations/checkpoint/actions.py

```diff
@@ -745,8 +753,8 @@ def __init__(
 
     def _run(
         self,
-        validation_result_suite,
-        validation_result_suite_identifier,
+        validation_result_suite: ExpectationSuiteValidationResult,
+        validation_result_suite_identifier: ValidationResultIdentifier,
```

Comment:
@roblim Same question as above: Do we need `Union` with `GeCloudIdentifier` here?  Thanks.